src/tests/tests.cpp

```diff
@@ -907,7 +976,7 @@ std::vector<Test::Result> Text_Based_Test::run()
             Test::Result result = run_one_test(header, vars);
             if(m_cpu_flags.size() > 0)
                {
-               for(auto&& cpuid_bit : m_cpu_flags)
+               for(auto && cpuid_bit : m_cpu_flags)
```

Comment:
No need for `auto&&` should be either `auto` or explicit `Botan::CPUID::CPUID_bits`